Assets/UI/worldviewiconsmanager_CQUI.lua

```diff
@@ -86,10 +91,10 @@ function SetResourceIcon( pInstance:table, pPlot, type, state)
   end
 end
 
+
 -- ===========================================================================
 --  CQUI modified AddImprovementRecommendationsForCity functiton
---  Don't show builder recommandation, it's often stupid
 -- ===========================================================================
 function AddImprovementRecommendationsForCity( pCity:table, pSelectedUnit:table )
-  return;
+	if CQUI_ShowImprovementsRecommendations then BASE_AddImprovementRecommendationsForCity(pCity, pSelectedUnit); end
```

Comment:
Style suggestion and not a blocking issue. My preference is to usually avoid if...then...end statement in a single line, I feel it makes it less readable if I'm trying to scan through code quickly. Do you know if the single line version is a common style for Lua?

Reply:
It is a matter of code readability, there is no such thing as "don't use single-line ifs" rule in Lua.
I 99.9% do use if-then-else on a separate lines except for cases like above - the condition is super simple and there is a simple call after that. That's it. Same thing as Firaxis does.
I would also like to point out that everyone has its own "style preferences" so please do not enforce your own on others unless it is something really important.

Reply:
This comment was not anything being enforced. I saw you doing something different from what I would do, so I mentioned why I do it my way and wanted to know why you do it your way. That is all. This would not have been a reason for not merging the pull, that's why I said it's not a blocking issue.

Reply:
@Infixo I am glad we now have 3 developers working on this mod, and I appreciate your help with the mod fixes. 
one thing I want to highlight - while we all have our preferences on code style, communication style and such, one the most important things to ensure a successful long-term working relationship is to be kind and respectful to each other, regardless of whether it is a commercial or a volunteering project. I expect that the areas of code ownership will be naturally split among the team over the next weeks or months, and I would be fine for the owner of each part to define his/her own local standards, given the non-commercial aspect of this project. 

I think having a half-hour video call for the whole team would be a good way to establish some human contact. what do you guys think? I can schedule something for the next weekend.